src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -40,6 +40,8 @@ public interface IPKAnalysesTask : OSPSuite.Core.Domain.Services.IPKAnalysesTask
 
       IndividualPKAnalysis CalculateFor(Simulation simulation, DataColumn dataColumn);
       PKValues CalculatePK(DataColumn column, PKCalculationOptions options);
+
+      PKAnalysis MapFrom(double? molWeight, PKValues pkValues, string moleculeName, Simulation simulation);
```

Comment:
Mapping typically does not belong in task but own mapper. Why is this different?

Reply:
It is actually wrapping the mapper. In addition, it uses the service to create the options which we do not have from the caller. So I could still use the mapper from the caller and ask only for the options to the service and then we have a better decoupling.